workflow/rules/authentic.smk

```diff
@@ -89,7 +92,7 @@ checkpoint Malt_Extract:
     message:
         "Malt_Extract: RUNNING MALT EXTRACT FOR SAMPLE {input.rma6}"
     shell:
-        "time MaltExtract -i {input.rma6} -f def_anc -o {params.extract} --reads --threads {threads} --matches --minPI 85.0 --maxReadLength 0 --minComp 0.0 --meganSummary -t {input.node_list} -v 2> {log}"
+        "time MaltExtract -Xmx32G -i {input.rma6} -f def_anc -o {params.extract} -r {params.ncbi_db} --reads --threads {threads} --matches --minPI 85.0 --maxReadLength 0 --minComp 0.0 --meganSummary -t {input.node_list} -v 2> {log}"
```

Comment:
Are we sure this works? I recall we removed this parameter some time ago, but now we seem to need it again?

Reply:
TJ reported that MaltExtract needs to increase the Java heap space for some taxa with lots of reads like humans. When I added this flag and tested on my laptop and Uppmax, it worked.

Reply:
I believe, the -r flag should be there by default because we have a rule NCBIMapTre that downloads ncbi.map and ncbi.tre and places them to ncbi_db. In case, the downloading fails, when there is not internet connection as TJ noticed, there should be at least a chance to point MaltExtract to ncbi_db where one could manually upload the ncbi.map and ncbi.tre files. I ran aMeta with the -r flag a few times on my laptop and Uppmax, i.e. with internet connection, and it always worked

Reply:
Ok, cool!